docs/content/docs/checks/AndroidLint.md

```diff
@@ -5,34 +5,42 @@ type: docs
 
 # Android Lint
 
-https://developer.android.com/studio/write/lint
+We use [Android lint](https://developer.android.com/studio/write/lint) to check applications in CI.
 
-В PR результаты отображаются во вкладках "\<app name\> lint".
+By default, we check only applications. It's no use to check libraries in isolation.
```

Comment:
```suggestion
By default, we check only applications. It's not use to check libraries in isolation.
```

Reply:
I mean useless: https://idioms.thefreedictionary.com/it%27s+no+use